src/eq_schema/schema/QuestionnaireFlow/index.js

```diff
@@ -0,0 +1,33 @@
+const { filter } = require("lodash");
+class QuestionnaireFlow {
+  constructor(questionnaireJson) {
+    this.type = questionnaireJson.hub ? "Hub" : "Linear";
+    this.options = this.buildOptions(questionnaireJson);
+  }
+
+  buildOptions(questionnaireJson) {
+    const options = {}
+    if(questionnaireJson.hub) {
+      const requiredCompletedSections =
+        this.buildRequiredSections(questionnaireJson);
+      if (requiredCompletedSections.length) {
+        options.requiredCompletedSections = requiredCompletedSections;
```

Comment:
```suggestion
        options.required_completed_sections = requiredCompletedSections;
```

`options.required_completed_sections` is part of Runner's JSON, so should be snake case.